subsys/bluetooth/host/conn.c

```diff
@@ -2399,6 +2416,8 @@ struct bt_conn *bt_conn_add_sco(const bt_addr_t *peer, int link_type)
 	} else if (link_type == BT_HCI_ESCO) {
 		sco_conn->sco.pkt_type = (bt_dev.br.esco_pkt_type &
 					  ~EDR_ESCO_PKT_MASK);
+	} else {
+		/* Ignoring unexpected link type BT_HCI_ACL. */
```

Comment:
I assume that this was added due to Sonarqube - Since it's just a comment it's OK, but otherwise just ignore it next time that Sonar complains about code you haven't modified.

Reply:
Yes, this was to fix a Sonarqube warning (and there's another similar change in the same file).
Ok, from now on I'll ignore it when it complains about code I didn't change, thanks.